include/whole_body_mpc_msgs/conversions.h

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright (c) 2021, University of Oxford
```

Comment:
Can we use a short format instead of the whole message?
This is the example
```c++
///////////////////////////////////////////////////////////////////////////////
// BSD 3-Clause License
//
// Copyright (C) 2021, University of Oxford
// Copyright note valid unless otherwise stated in individual files.
// All rights reserved.
///////////////////////////////////////////////////////////////////////////////
```

We should also add the include the LICENSE file.